tests/local/cheyenne/model_test.sh

```diff
@@ -175,7 +176,8 @@ module list
 #-------------------------------------------------------
 # Python Env
 deactivate > /dev/null 2>&1
-source /glade/p/cisl/nwc/model_testing_env/wrf_hydro_nwm_test/bin/activate || exit 9
+# source /glade/p/cisl/nwc/model_testing_env/wrf_hydro_nwm_test/bin/activate || exit 9
+source /glade/work/jamesmcc/python_envs/36dv/bin/activate || exit 9
```

Comment:
Can this be moved into a /glade/p/cisl/nwc location instead of a personal directory?

Reply:
🤦‍♂ 
that was my intent, i really bungled this PR. I will fix it

Reply:
um, yea, I did fix it. I belive you were looking at stale diffs